test/output/slim/block_admonition.html

```diff
@@ -0,0 +1,72 @@
+<!-- .basic -->
+<div class="admonitionblock warning">
+  <table>
+    <tr>
+      <td class="icon">
+        <div class="title">Warning</div>
+      </td>
+      <td class="content">Watch out for dragons!</td>
+    </tr>
+  </table>
+</div>
+
+<!-- .basic_multiline -->
```

Comment:
This file is outdated, I changed names format in beta.3 to use hyphen instead of underscore.